src/frontend/screens/Settings/MapSettings/BackgroundMapInfo.tsx

```diff
@@ -163,24 +173,43 @@ export const BackgroundMapInfo = ({
         return allSettled(tileSetIds.map(id => api.maps.getTileset(id)));
       })
       .then(tileJson => {
-        const maxZoom = tileJson.reduce((highest, tilePromise) => {
-          if (!("value" in tilePromise)) return highest;
-          const tile = tilePromise.value;
-          if (!tile.maxzoom) return highest;
-          if (tile.maxzoom > highest) return tile.maxzoom;
-          return highest;
-        }, -1);
+        const initialVal: (undefined | number)[] = [undefined, undefined];
+        const zoomRanges: (undefined | number)[] = tileJson.reduce(
```

Comment:
there's probably a simpler way of calculating this using `Math.min` and `Math.max`. Can't suggest something off the top of my head but basically collecting all of the min zooms and max zooms, and passing them to the math functions to get the full range